layers/+emacs/emacs-info/README.org

```diff
@@ -0,0 +1,10 @@
+#+TITLE: Emasc Info Imrprovements
+
```

Comment:
You are missing a table of content which is necessary for generating HTML documents from org.
```suggestion
* Table of Contents                     :TOC_5_gh:noexport:
- [[#description][Description]]
  - [[#features][Features:]]
- [[#install][Install]]
- [[#key-bindings][Key bindings]]
  - [[#transient-state][Transient state]]
```